### ISSUE TITLE
Nuplan or NuPlan or nuPlan?

### ISSUE THREAD

**Issue by pengzhenghao** [metadriverse/metadrive#372]
Change the wording

**Comment by QuanyiLi** [comment]
NuPlan is good. Just follow the official name

**Comment by pengzhenghao** [comment]
The official use nuPlan.Best,ZhenghaoOn Mar 27, 2023, at 02:22, Quanyi Li ***@***.***> wrote:﻿
Closed #372 as completed.

**Comment by QuanyiLi** [comment]
No. In nuplan-devkit, people use NuPlan, like ```class NuPlanScenario```

**Comment by pengzhenghao** [comment]
great then NuPlan. Check and transform code in our project before closing this issue.

**Comment by QuanyiLi** [comment]
All class name containing nuplan is ```NuPlan```, in other places it is ```nuplan```. Close this issue.
